### ISSUE TITLE
publish new release

### ISSUE THREAD

**Issue by wbolster** [mitsuhiko/pipsi#36]
pipsi with python 3 support from #21 is not installable via `pip` currently (which the `get-pipsi` script uses), which means that we get to miss out on all the python3 fun.

please publish a new release. :)

**Comment by sscherfke** [comment]
+1

**Comment by wbolster** [comment]
In the mean time, you can use this to upgrade your local pipsi install to the current `master` branch:

```
cd ~/.local/venvs/pipsi
./bin/pip uninstall pipsi
git clone git@github.com:mitsuhiko/pipsi.git src
cd src
../bin/pip install -e .
```

This will change the virtualenv that pipsi uses by replacing the installed version with a development version.

**Comment by jhermann** [comment]
`~/.local/venvs/pipsi/bin/pip install -UI https://github.com/mitsuhiko/pipsi/archive/master.zip`

Shorter, and doesn't require git installed.

**Comment by wbolster** [comment]
@mitsuhiko what can we do to bribe you into uploading a new version? :)

**Comment by RonnyPfannschmidt** [comment]
various pr's need to be merged first

**Comment by ryneeverett** [comment]
@RonnyPfannschmidt Are any of the remaining PR's essential?

**Comment by RonnyPfannschmidt** [comment]
work is further on, waiting for pypi perms now

**Comment by stefanfoulis** [comment]
@mitsuhiko any chance we could release on pypi?

**Comment by sanjioh** [comment]
+1 :)
It would be nice to have a new release of pipsi.
Is there anything we can do to make this happen?
